tests/factories/reports/cht/pregnancy.js

```diff
@@ -0,0 +1,314 @@
+    '__tested_for_hiv_in_past_3_months': 'no',
+    '__received_tetanus_toxoid_this_pregnancy': 'no',
+    'meta': {
+      '__patient_uuid': '80b6e34e-7abf-5718-a572-84edbcd4593e',
+      '__patient_id': '10072',
+      '__household_uuid': '3a9cecea-8414-5768-9846-474d21b51e91',
+      '__source': 'contact',
+      '__source_id': ''
+    }
+  },
+  'meta': {
+    'instanceID': 'uuid:09318528-a484-4d9d-b49c-473b84c22ae9'
+  }
+};
+
+Factory.define('basePregnancy')
+  .sequence('_id',uuid.v4)
+  .attr('form', 'pregnancy')
+  .attr('type', 'data_record')
+  .attr('content_type', 'xml')
+  .attr('reported_date', Date.now())
+  .attr('contact', 'TODO')
+  .attr('from', '')
+  .attr('hidden_fields', [
+    'patient_age_in_years',
+    'patient_uuid',
+    'patient_id',
+    'patient_name',
+    'patient_short_name',
+    'patient_short_name_start',
+    'lmp_date_8601',
+    'edd_8601',
+    'days_since_lmp',
+    'weeks_since_lmp',
+    'weeks_since_lmp_rounded',
+    'lmp_method_approx',
+    'hiv_status_known',
+    'deworming_med_received',
+    'tt_received',
+    't_pregnancy_follow_up_date',
+    't_pregnancy_follow_up',
+    't_danger_signs_referral_follow_up_date',
+    't_danger_signs_referral_follow_up',
+    'gestational_age.register_method.register_note',
+    'gestational_age.method_approx.lmp_approx',
+    'gestational_age.method_lmp_summary',
+    'gestational_age.g_lmp_date_8601',
+    'gestational_age.g_lmp_date',
+    'gestational_age.g_edd_8601',
+    'gestational_age.g_edd',
+    'anc_visits_hf.anc_visits_hf_past.visited_dates_group.visited_count_confim_note_single',
+    'anc_visits_hf.anc_visits_hf_next.anc_visit_advice_note',
+    'risk_factors.r_risk_factor_present',
+    'danger_signs.danger_signs_note',
+    'danger_signs.r_danger_sign_present',
+    'danger_signs.congratulate_no_ds_note',
+    'safe_pregnancy_practices.malaria.llin_advice_note',
+    'safe_pregnancy_practices.malaria.malaria_prophylaxis_note',
+    'safe_pregnancy_practices.iron_folate.iron_folate_note',
+    'safe_pregnancy_practices.safe_practices_tips',
+    'safe_pregnancy_practices.hiv_status.hiv_importance_note',
+    'safe_pregnancy_practices.tetanus.tt_note_1',
+    'safe_pregnancy_practices.tetanus.tt_note_2',
+    'safe_pregnancy_practices.request_services',
+    'summary',
+    'data',
+    'meta'
+  ])
+  .attr('geolocation_log', [
+    {
+      'timestamp': 1618841239430,
+      'recording': {
+        'latitude': 39.9432946,
+        'longitude': -82.9788323,
+        'altitude': null,
+        'accuracy': 1518,
+        'altitudeAccuracy': null,
+        'heading': null,
+        'speed': null
+      }
+    }
+  ])
+  .attr('geolocation',  {
+    'latitude': 39.9432946,
+    'longitude': -82.9788323,
+    'altitude': null,
+    'accuracy': 1518,
+    'altitudeAccuracy': null,
+    'heading': null,
+    'speed': null
+  });
+
+
+Factory.define('pregnancy').extend('basePregnancy').attr('fields', defaultFields);
```

Comment:
This would allow us to define different pregnancies. 

EX:

`Factory.define('pregnancyWithRiskFactors').extend('basePregnancy').attr('fields', riskFactorFields);`